explore/github-data/labRepos_Activity.json

```diff
@@ -24409,6 +24409,19 @@
             }
         ],
         "CDAT/ci-bots": [
+            {
```

Comment:
@LRWeber - Is there anything that we can do to order these json sub-objects, to try to reduce the diffs? 

I think in this block the only line that REALLY changed is the ordering based on the `week` value in the object... Is that something that we are able to do?

Reply:
In these updates, all of the changes are either

1. Activity count changes for the active week (or past weeks I suppose if older commits get merged in)
2. An entire week being removed or added based on the shifting timeline bounds

The numbers should differ from week to week. In this case, both weeks were inactive, so yes the data is virtually identical, but this is rarely the case for active repos. Additionally, the data is supposed to be a date-ordered-array, which means in each case, one date is at the beginning and one is at the end.

However... The way we use the data currently, we've only been utilizing the weekly totals. If we don't expect to drill down into any more detailed views on our own portal (especially since interested parties can always check a repo's GitHub page as well), we _could_ potentially strip out the `days` arrays completely, which would significantly shrink the file and the diffs.

Reply:
Ahh... That helped me see what I was missing... I was thinking these changes (addition and deletion) were all this year, but they are differnet years... That's why it looks "not ordered" to me...

Reply:
Should we strip the daily counts anyway? It's a huge chunk of data that we aren't using, and it'd be a one-line fix.

Reply:
If we aren't using it, yeah, let's do that.

Maybe just comment it out for now, with a comment pointing to this conversation? Then if we need it in the future it's not too tough to add back?